example/scripts/tests-parallel.sh

```diff
@@ -14,7 +14,7 @@ testOutputDir=scripts/test_output
 rm -r -f $testOutputDir
 mkdir -p $testOutputDir
 
-echo "" | ../node_modules/.bin/parallel -j 0 --timeout 30000 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
+echo "" | ../node_modules/.bin/parallel -j 0 --timeout 1200 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
```

Comment:
I had actually done this wrong and fixed it here. The timeout now is 1200 seconds for a command, which is an individual test run.